cabal-install/changelog

```diff
@@ -2,6 +2,7 @@
 
 2.6.0.0 (current development version)
 	* New solver flag: '--reject-unconstrained-dependencies'. (#2568)
+        * New v2-build flag: '--only-configure'. (#5578)
```

Comment:
tab vs spaces :S

Reply:
The entry just below `Bugfix: "cabal new-build --ghc-option '--bogus' --ghc-option '-O1'"` also has spaces, I just copied that one ;)